.github/workflows/post-apiview.yml

```
@@ -0,0 +1,29 @@
+name: After APIView
+
+on:
+  check_run:
+    types: [completed]
+    
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  event-handler:
+    name: After APIView
+    runs-on: ubuntu-24.04
+    if: ${{ github.event.check_run.check_suite.app.name == 'Azure Pipelines' && 
+          ( contains(github.event.check_run.name, 'APIView') ||
+            contains(github.event.check_run.name, '(SDK Generation') ||
+            contains(github.event.check_run.name, 'Azure.azure-rest-api-specs-pipeline') ) }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: 'eng/common'
+    
+      - name: Create APIView Comment on PR
+        run: |
+          . ../../../eng/common/scripts/helpers/ApiView-Helpers.ps1
+          Set-ApiViewCommentForRelatedIssues -HeadCommitish ${{ github.event.check_run.head_sha }} -AuthToken ${{ secrets.GITHUB_TOKEN }}
```

For security, I think it's preferred to pass `secrets.GITHUB_TOKEN` via env var.  I think it's easier to accidentally leak the token if you pass it on the command line.  Example:
```suggestion
          Set-ApiViewCommentForRelatedIssues -HeadCommitish ${{ github.event.check_run.head_sha }}
        env:
          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-github-cli-in-workflows